### PR TITLE
[sc-22364] GitHub template part 1

### PR DESCRIPTION
**Pull request by arooxa**
This template has all the github MVP entities without the relationships defined. 

Relationships will be added in this follow-up [PR](https://github.com/SGNL-ai/catalog-sgnl-templates/pull/44) - The follow-up template PR with defined relationships is waiting on the UI changes in this [PR](https://app.shortcut.com/sgnl/story/22363/fe-catalog-defining-relationships-limitations)

### CODE REVIEW

File: github.sgnl.yaml [hunk 1]
```diff
@@ -0,0 +1,700 @@
...
+        externalId: $.node.id
+        type: String
+        indexed: true
+        uniqueId: true
+        attributeAlias: repositoryCollaboratorId
+      - name: permission
+        externalId: permission
+        type: String
+  # Collaborator is a User entity type: https://docs.github.com/en/graphql/reference/objects#repositorycollaboratoredge
+  Collaborator:
```
Comment (r-rakshith) [inline github.sgnl.yaml:553]:
```suggestion

  # Collaborator is a User entity type: https://docs.github.com/en/graphql/reference/objects#repositorycollaboratoredge
  Collaborator:
```
nit: Add new line between all entities

File: github.sgnl.yaml [hunk 2]
```diff
@@ -0,0 +1,700 @@
...
+      - name: viewerCanCreateProjects
+        externalId: viewerCanCreateProjects
+        type: Bool
+      - name: viewerCanFollow
+        externalId: viewerCanFollow
+        type: Bool
+      - name: viewerIsFollowing
+        externalId: viewerIsFollowing
+        type: Bool
+relationships:
```
Comment (r-rakshith) [inline github.sgnl.yaml:641]:
```suggestion
        type: Bool
    
relationships:
```
nit: Add new line between entity and relationship

**Review by r-rakshith — APPROVED** [review]
LGTM

File: github.sgnl.yaml (hunk 2 above)
Comment (arooxa) [reply github.sgnl.yaml:641]: I'll address this one in the follow-up PR

File: github.sgnl.yaml (hunk 1 above)
Comment (arooxa) [reply github.sgnl.yaml:553]: None of the other templates have new lines between entities so I'll keep that format unless you want to change this

**Pull request merged**